packages/types/src/primitive/StorageKey.spec.ts

```diff
@@ -123,4 +124,32 @@ describe('StorageKey', () => {
       ).toEqual(Uint8Array.from([1, 1, 48, 7, 201, 255, 112, 39, 246, 89, 0, 171, 205, 252, 164, 253, 177, 7, 234, 212, 126, 42, 158, 53, 88, 224, 27, 105, 27, 15, 74, 95, 133, 24, 212, 135, 50, 102, 20, 240, 102, 65, 99, 8, 191, 106, 164, 229, 4, 29, 25, 73, 146, 142, 75, 38, 237, 233, 142, 60, 235, 179, 106, 59, 23, 38]));
     });
   });
+
+  describe('with MetadataV6', () => {
```

Comment:
These tests should reflect what changed in this metadata version. In my v5 PR i just copied the  two double map tests because `EventTopics` is the only entry in the metadata v5 (and v4) that uses `DoubleMapType` and the changes in v5 were related to `DoubleMapType`.

Here we should test if the values of some of the modules that are using `parameter_types` now are added correctly.

Reply:
But feel free to merge and put it in a second PR! I know  I'm a little late to the party with my comment here

Reply:
AFAICT, storageKey hasn't changed between v5->v6 so i just copied the same test, i.e. should expect the same result with v6 as with v5.

Reply:
Ah, ok, makes sense!